File: js/lv1/문자열다루기기본.js
function solution(s) {
  return (
    parseInt(s) == s && (s.length === 4 || s.length === 6)
  );
}

strings = ["a234", "1234", "2e11"];
answer = [false, true, false];

for (let i = 0; i < strings.length; i++) {
  if (solution(strings[i]) == answer[i]) {
    // console.log(`${i + 1}: pass`);
  } else {
    // console.log(`${i + 1}: fail`);
  }
}

File: python/lv1/문자열다루기기본.py
def solution(s):
    if len(s) != 4 and len(s) != 6:
        return False
    for ss in s:
        if not ss.isdigit():
            return False
    return True


strings = ['a234', '1234']
answer = [False, True]

for i in range(len(strings)):
    if solution(strings[i]) == answer[i]:
        print(f'{i+1}: pass')
    else:
        print(f'{i+1}: fail')

# def alpha_string46(s):
#     return s.isdigit() and len(s) in (4, 6)